22_pipeslab/pipeslab.txt

```diff
@@ -1,21 +1,23 @@
 For each of the following, type a single command line that performs:
 
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
-  Answer: 
+  Answer: ls -l | cut -c4 | grep x | wc | tr -s ' ' | cut -d" " -f2
 
   2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
-  Answer: 
+  Answer: ls -l | cut -c4,7,10 | grep xxx | wc | tr -s ' ' | cut -d" " -f2
 
   3. Print the number of lines from "ls -l" which represent paths that were modified today (no matter which day we run the command)
-  Answer: 
+  Answer: ls -l --time-style=full-iso| grep $(date +%Y-%m-%d) | wc | tr -s ' ' | cut -d" " -f2
 
   4. Modify (1) and (2) to count files only (no directories)
   Answer:
+  ls -l | cut -c1,4 | grep -x | wc | tr -s ' ' | cut -d" " -f2
+  ls -l | cut -c1,4,7,10 | grep -xxx | wc | tr -s ' ' | cut -d" " -f2
```

Comment:
I assume that ```grep -xxx``` & ```grep -x``` are copy-paste errors?
you can pass regex to grep or you can use grep with ```-v``` flag